hack/install-vm.sh

```diff
@@ -223,19 +470,28 @@ esac
 done
 
 # Check whether OVS configuration needs to be cleaned up.
-if [ "$VALIDATE_OVS_CONFIG" = true ] ; then
+if [ "$VALIDATE_OVS_CONFIG" = true ] && [ -z "$DOCKERIZE" ]; then
     check_ovs_config_and_cleanup
     exit 0
 fi
 
-# Check for mandatory arguments.
-if [ -z "$AGENT_BIN_PATH" ] || [ -z "$CONFIG_PATH" ] || [ -z "$KUBECONFIG" ] || [ -z "$ANTREA_KUBECONFIG" ] ; then
-    echoerr "Missing argument(s)"
-    print_usage
-    exit 1
-fi
+validate_mandatory_arguments() {
+    if [ -z "$CONFIG_PATH" ] || [ -z "$KUBECONFIG" ] || [ -z "$ANTREA_KUBECONFIG" ] || ( [ -z "$AGENT_BIN_PATH" ] && [ -z "$DOCKERIZE" ] ); then
+        echoerr "Missing argument(s)"
+        print_usage
+        exit 1
+    fi
+}
 
+validate_mandatory_arguments
 check_supported_platform
 copy_antrea_agent_files
 update_antrea_agent_conf
-start_antrea_agent_service
+if [ -z "$DOCKERIZE" ]; then
+    start_antrea_agent_service
+else
+    install_docker_requisite_packages
```

Comment:
Did you test this version of the script? 
Looks like you missed renaming this function.
```suggestion
    install_docker_prerequisite_packages
```